keras/utils/data_utils.py

```diff
@@ -214,7 +214,7 @@ def dl_progress(count, block_size, total_size):
             if os.path.exists(fpath):
                 os.remove(fpath)
             raise
-        progbar = None
+        progress_tracker.progbar = None
```

Comment:
Is it necessary to reset the progbar to None, or should this line just be removed?